src/tensorwaves/optimizer/minuit.py

```diff
@@ -73,7 +73,7 @@ def wrapped_function(pars: list) -> float:
             update_parameters(pars)
             parameters = parameter_handler.unflatten(flattened_parameters)
             estimator_value = estimator(parameters)
-            progress_bar.set_postfix({"estimator": estimator_value})
+            progress_bar.set_postfix({"estimator": float(estimator_value)})
```

Comment:
Why is this cast needed? Is the output from tf not pretty?

Reply:
Yes, the estimator value is a `Tensor`, that looked weird in the progress bar